### DISCUSSION TITLE
does not exist in the schema for Ring-MQTT with Video Streaming

### DISCUSSION THREAD

**Discussion by Captain0Kangaroo** [tsightler/ring-mqtt#382] [General Discussion]
I'm getting these errors in my logs an not sure why or how to fix them. Any ideas?

22-05-30 08:31:41 WARNING (MainThread) [supervisor.addons.options] Option 'snapshot_mode' does not exist in the schema for Ring-MQTT with Video Streaming (03cabcc9_ring_mqtt)
22-05-30 08:31:41 WARNING (MainThread) [supervisor.addons.options] Option 'beam_duration' does not exist in the schema for Ring-MQTT with Video Streaming (03cabcc9_ring_mqtt)
22-05-30 08:31:41 WARNING (MainThread) [supervisor.addons.options] Option 'mqtt_host' does not exist in the schema for Ring-MQTT with Video Streaming (03cabcc9_ring_mqtt)
22-05-30 08:31:41 WARNING (MainThread) [supervisor.addons.options] Option 'mqtt_port' does not exist in the schema for Ring-MQTT with Video Streaming (03cabcc9_ring_mqtt)
22-05-30 08:31:41 WARNING (MainThread) [supervisor.addons.options] Option 'mqtt_user' does not exist in the schema for Ring-MQTT with Video Streaming (03cabcc9_ring_mqtt)
22-05-30 08:31:41 WARNING (MainThread) [supervisor.addons.options] Option 'mqtt_password' does not exist in the schema for Ring-MQTT with Video Streaming (03cabcc9_ring_mqtt)
22-05-30 08:31:41 WARNING (MainThread) [supervisor.addons.options] Option 'ring_token' does not exist in the schema for Ring-MQTT with Video Streaming (03cabcc9_ring_mqtt)

**Comment by tsightler** [comment]
Assuming you upgraded from 4.x to 5.x, this is covered in the Upgrading to 5.x documentation.  Basically, these options are no longer valid configuration options and you should manually remove them from the configuration and save the new configuration without them, however, they should not cause any issues if they are still there, as this is just a warning message that they are no longer used.

**Comment by thebatfink** [comment]
How does one remove these configuration options? I don’t have them set in the addon configuration window (blank values) already. I tried just saving the configuration again with the fields blank (although they aren’t the same fields are they) and still the logs become populated with these every 5 minutes. There is a file with them saved somewhere? (I never set anything in configuration.yaml).

**Reply by tsightler** [reply]
Weird, I'm not sure why it would show up every 5 minutes, maybe some side effect of the "Repairs" feature introduced a few months back.  I wouldn't have expected these to show up except during a restart of the addon.

Regardless, when I state that you should remove them from the configuration, I'm referring to the configuration tab of the addon.  If you click the three dots and select "Edit in YAML" you can just remove the unused lines from the configuration.  I actually thought you could delete them from the UI configuration mode as well as I thought unused configuration options displayed with an X that could be used to remove them, but I don't have any way to test this right now.

**Reply by thebatfink** [reply]
Edit in YAML was what I was missing. I see the settings now, removed them and all good. Thanks!
